security.rst

```diff
@@ -1602,6 +1602,20 @@ and set the ``limiter`` option to its service ID:
             ;
         };
 
+Customize Successful And Failed Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Should we add something like "behavior" / "handling"

`Customize Successful And Failed Authentication Behavior`
```

Reply:
Good idea, updated :slightly_smiling_face: